epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModelGroup.java

```diff
@@ -57,11 +58,14 @@
  * manage their children. The shared pool is cleaned up when the activity is destroyed.
  */
 @SuppressWarnings("rawtypes")
-public class EpoxyModelGroup extends EpoxyModelWithHolder<ModelGroupHolder> {
+public class EpoxyModelGroup extends EpoxyModelWithHolder<ModelGroupHolder> implements ModelCollector {
 
-  protected final List<? extends EpoxyModel<?>> models;
-  /** By default we save view state if any of the models need to save state. */
-  private final boolean shouldSaveViewState;
+  protected final List<EpoxyModel<?>> models;
+
+  private boolean shouldSaveViewStateDefault = false;
+
+  @Nullable
+  public Boolean shouldSaveViewState = null;
```

Comment:
should this not be public? do we want to force usage of `shouldSaveViewState` so the mutation check is always done?

Reply:
Good catch, I can't remember why I changed it to `public`